guardian/support-frontend
# Refactor SSR content rendering into one script

## What are you doing in this PR?

Refactoring the way we create server side rendered content into one script, so it will be easier to add more new pages.



## Why are you doing this?

We added a new page in #6412, this tidies things up a bit.



## How to test

Run `yarn build-ssr`, go to `/newspaper-archive-header` or `/uk/contribute` and see loading component.
